### PR TITLE
fixed display function

### LINKED ISSUE #1: Display
## 1. Fix `display.js`

__As a site visitor, I want to execute the function `add-number.js`__

### Repo

- [x] developed on a branch named `display`

### JS

- [x] fix the script `display.js`

### PR DESCRIPTION
**Pull request by yildiraykoyuncu**
close #1

### CODE REVIEW

**Comment by aleks2407** [comment]
Good job :) the only one thing , in demo word "All" is not capitalized.

File: scripts/user-actions/display.js [hunk 1]
```diff
@@ -3,14 +3,14 @@ console.log('--- loading: display-all.js');
...
 //   the handler has no tests, study the demo to write this function
-const displayHandler = (message = '', arrayToDisplay = words) => {
-  debugger;
-  const combineStrings = (acc, next) => {
-    return _;
-  }
-  const alertMessage = arrayToDisplay
-    ._(_, _);
-  alert(_);
+const displayHandler = (message = 'All words:', arrayToDisplay = words) => {
```
Comment (aleks2407) [inline scripts/user-actions/display.js]: should be `all words:`

**Review by aleks2407 — CHANGES_REQUESTED** [review]

**Review by abelRoland — APPROVED** [review]
Good job! Thanks to do it!

**Review by abelRoland — APPROVED** [review]

**Review by aleks2407 — APPROVED** [review]

**Review by Feruzteame — APPROVED** [review]

**Comment by Feruzteame** [comment]
nice !

**Pull request merged**